test/Test-Mathilde.js

```diff
@@ -61,6 +73,43 @@ describe('SimpleNftMarketplace', function () {
       expect(await contract.connect(lucie).isModerator(lucie.address)).to.be.false;
       expect(await contract.connect(nonModerator).isModerator(nonModerator.address)).to.be.false;
     });
+
+    it('Should add a role to an account', async function () {
+      const moderatorRole = contract.MODERATOR_ROLE();
+      const adminRole = contract.DEFAULT_ADMIN_ROLE();
```

Comment:
Add assertion before you grant role, to confirm the user does not have the role before